Sources/hostmgr/SyncCommand.swift

```diff
@@ -45,8 +45,12 @@ struct SyncCommand: ParsableCommand {
 
     private func perform(task: Configuration.SchedulableSyncCommand, immediately: Bool) throws {
         switch task {
-        case .authorizedKeys: try SyncAuthorizedKeysTask().run(force: immediately)
-        case .vmImages: try SyncVMImagesTask().run(force: immediately)
+        case .authorizedKeys:
+            let command = SyncAuthorizedKeysCommand(force: immediately)
+            try command.run()
+        case .vmImages:
+            let command = SyncVMImagesCommand(force: immediately)
+            try command.run()
```

Comment:
Thanks for isolating this!! Resolved with better option hierarchy in 9e4a1c3.

Reply:
FWIW a slightly faster way to do this is to use Xcode launch arguments, in case it helps!

<img width="1881" alt="Screen Shot 2022-10-24 at 6 58 45 PM" src="https://user-images.githubusercontent.com/1123407/197657711-20838d29-c6ef-4f33-a0ff-34320ba91a97.png">

Reply:
Neat! Thanks!